knative-operator/pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -323,6 +331,13 @@ func (r *ReconcileKnativeServing) installQuickstarts(instance *operatorv1beta1.K
 
 // installKnConsoleCLIDownload creates CR for kn CLI download link
 func (r *ReconcileKnativeServing) installKnConsoleCLIDownload(instance *operatorv1beta1.KnativeServing) error {
+	// Skip installing console cli download if there are no related CRDs available eg. cluster is installed without console
+	if _, err := r.apiExtensionV1Client.CustomResourceDefinitions().Get(context.Background(), "consoleclidownloads.console.openshift.io", metav1.GetOptions{}); err != nil {
+		if errors.IsNotFound(err) {
+			return nil
+		}
+		return fmt.Errorf("failed to fetch ConsoleCLIDownload CRDs: %w", err)
+	}
```

Comment:
We talked with @rhuss briefly when the original issue was raised. That maybe we should only skip creation of `ConsoleCliDownloads` resource, but keep the download route itself. Nested in the `.Apply()` function. Therefore download URL could be easily inferred with enougt insight about it. 

https://github.com/openshift-knative/serverless-operator/blob/main/knative-operator/pkg/controller/knativeserving/consoleclidownload/consoleclidownload.go#L29-L36

Reply:
@dsimansk Without the Console, the download link will not work.